src/__tests__/linter.test.ts

```diff
@@ -206,19 +206,16 @@ responses:: !!foo
     expect(result).toEqual([
       expect.objectContaining({
         code: 'oas3-schema',
-        summary: 'Validate structure of OpenAPIv3 specification.',
```

Comment:
If `summary` is the same as `message`, wouldn't it be worth to still check its presence?

Reply:
Sounds reasonable.